tests/model_serving/maas_billing/oidc_tests/conftest.py

```
@@ -0,0 +1,405 @@
+    )
+    credential_data = credentials_secret.instance.data
+
+    user_names = base64.b64decode(credential_data.users).decode().split(",")
+    passwords = base64.b64decode(credential_data.passwords).decode().split(",")
+
+    assert user_names and user_names != [""], "No usernames found in byoidc-credentials secret"
+    assert passwords and passwords != [""], "No passwords found in byoidc-credentials secret"
+
+    non_admin_users = [
+        (username, password)
+        for username, password in zip(user_names, passwords)
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)